.github/workflows/pull_requests.yaml

```diff
@@ -0,0 +1,132 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Pull Request Checks
+
+on:
+  pull_request:
+    branches:
+      - "**"
+    paths:
+      - .buildkite/*
+      - .github/*.sh
+      - .github/ct-redpanda.yaml
+      - .github/kind.yaml
+      - .github/workflows/pull_requests.yaml
+      - charts/**
+      - 'charts/console/**'
+      - 'charts/redpanda/**'
+      - '!charts/connectors/**'
+      - '!charts/kminion/**'
+      - '!charts/operator/**'
+      - '!**/*.md'
+
+defaults:
+  run:
+    shell: nix develop --impure --command bash {0}
+
+jobs:
+  changes:
+    runs-on: ubuntu-22.04
+    outputs:
+      charts-connectors: ${{ steps.chart-changes.outputs.charts-connectors }}
+      charts-console: ${{ steps.chart-changes.outputs.charts-console }}
+      charts-kminion: ${{ steps.chart-changes.outputs.charts-kminion }}
+      charts-operator: ${{ steps.chart-changes.outputs.charts-operator }}
+      charts-redpanda: ${{ steps.chart-changes.outputs.charts-redpanda }}
+      go-code: ${{ steps.chart-changes.outputs.go-code }}
+    steps:
+    - uses: actions/checkout@v4
+    - uses: dorny/paths-filter@v3
+      id: chart-changes
+      with:
+        filters: |
+          charts-redpanda:
+            - 'charts/redpanda/**'
+          charts-console:
+            - 'charts/console/**'
+          charts-connectors:
+            - 'charts/connectors/**'
+          charts-kminion:
+            - 'charts/kminion/**'
+          charts-operator:
+            - 'charts/operator/**'
```

Comment:
If this is included then lines 31-33 should changed, right? There should not be negation in front of the path.

https://github.com/redpanda-data/helm-charts/pull/1112/files#diff-33cb58679656b227d577808dd88a6d7a9a915d63628459c99181089d3c5b9978R31-R33

Reply:
Nice catch, fixed!